### PR TITLE
Proofread SP Installation guide 

### PR DESCRIPTION
**Pull request by quinntracy**

### CODE REVIEW

File: content/en/docs/refguide/installation/install.md [hunk 1]
```diff
@@ -14,47 +14,39 @@ aliases:
...
 ## 2 Downloading Mendix Studio Pro
 
-Mendix Studio Pro can be installed on your Windows machine with a Windows executable file or on your Mac machine using a pkg file. This file can be downloaded from the Mendix Marketplace. 
+Mendix Studio Pro can be installed on your Windows machine with a Windows executable file or on your Mac machine using a .pkg file. This file can be downloaded from the Mendix Marketplace. 
 
 Follow these steps to download Mendix Studio Pro:
 
 1. Go to the Studio Pro download page in the [Mendix Marketplace](https://marketplace.mendix.com/link/studiopro/).
-2. Click the **Download** button in the upper-right corner to download the latest Mendix Studio Pro. Make sure to download the version for your operating system. 
+2. Click the **Download** button to download- Mendix Studio Pro. Make sure to download the correct version for your operating system.
```
Comment (quinntracy) [inline content/en/docs/refguide/installation/install.md]: Should "Download" be bolded, as there is no specific "Download" UI and rather asks which installer to download (Windows or Mac)?

File: content/en/docs/refguide/installation/install.md [hunk 2]
```diff
@@ -14,47 +14,39 @@ aliases:
...
 
     {{< figure src="/attachments/refguide/installation/install/setup-wizard.png"  class="no-border" >}}
 
-     On a Mac, you may get a warning during installation. For more information on how to troubleshoot it, see the [Mac Warning](#mac-warning) section below.
+     On a Mac, you may get a warning during installation. For more information on how to troubleshoot this warning, see the [Mac Warning](#mac-warning) section below.
 
 2. Select **I accept the terms in the License Agreement** and click **Next**.
 
-3. Select the folder in which you want to install Studio Pro and click **Next**:
+3. Check the desktop option to create a shortcut to Studio Pro on your desktop and click **Next**.
```
Comment (quinntracy) [inline content/en/docs/refguide/installation/install.md:46]: I verified the current steps and there is no longer the option to choose a folder to install SP in, so I removed those specific steps

File: content/en/docs/refguide/installation/install.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/installation/install.md]: Good catch, I think when Studio Pro on Mac was introduced, they changed the **Download** button into two, perhaps it is worth rewritting it to something like, click **Download on Windows** or **Download on Mac** (in Beta). Or: "Choose between downloading Studio Pro on Windows or Mac (in Beta) depending on your operating system."

Comment (quinntracy) [reply content/en/docs/refguide/installation/install.md]: I like the former, so I'll update to that. Does it need to say "in beta" or should it be based on the current UI?

File: content/en/docs/refguide/installation/install.md [hunk 3]
```diff
@@ -14,57 +14,49 @@ aliases:
...
 
 ### 4.1 Installing Prerequisites Separately {#prerequisites}
 
-Sometimes you can run into problems when installing Studio Pro. One work-around is to restart your system and install the prerequisites separately if they are not installed yet. 
+If you run into problems installing Studio Pro, one work-around is to restart your system and install the prerequisites separately if they are not installed yet. 
 
 The prerequisites are the following:
 
-* [Microsoft .NET Desktop Runtime 6.0.x](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) – Mendix recommends using version 6.0.6 or above
+* [Microsoft .NET Desktop Runtime 6.0.x](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) – It is recommended to use version 6.0.6 or above.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/installation/install.md]: It's in our style guide that we can use "Mendix recommends", but I guess "It is reccommended" is also fine, but I would not remove replace it everywhere.

File: content/en/docs/refguide/installation/install.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/installation/install.md]: We can leave "in beta" out indeed, it can be based on the current UI.

**Comment by MariaShaposhnikova** [comment]
Hi Quinn,
I left a small general comment, the PR looks good!

**Pull request merged**